docs/developer-onboarding.md

```diff
@@ -4,6 +4,12 @@
 
 This document describes the on-boardng steps for new Developers when they join the team.
 
+## You can request access to the following by asking in the #digital-tools-support slack channel:
+
+- [DfE-digital](https://github.com/DFE-Digital) GitHub group access. This will give you access to all repos but ours are: [Find](https://github.com/DFE-Digital/find-teacher-training) and [Apply](https://github.com/DFE-Digital/apply-for-teacher-training)
```

Comment:
```suggestion
- [DfE-Digital](https://github.com/DFE-Digital) GitHub group access. This will give you access to all repos. Ours are [Find](https://github.com/DFE-Digital/find-teacher-training) and [Apply](https://github.com/DFE-Digital/apply-for-teacher-training)
```